### PR TITLE
Reuse cluster fix for test stats creation, in case if stats is already there

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/dXMdraa8/1438-test-is-failing-if-teststats-were-already-created
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/es.py [hunk 1]
```diff
@@ -34,7 +34,10 @@ def create_doc(self, index, doc_type, doc_id, body):
         LOGGER.info('DOC_ID: %s', doc_id)
         LOGGER.info('BODY: %s', body)
         self._create_index(index)
-        self.create(index=index, doc_type=doc_type, id=doc_id, body=body)
+        if self.exists(index=index, doc_type=doc_type, id=doc_id):
+            self.update(index=index, doc_type=doc_type, id=doc_id, body={'doc': body})
```
Comment (amoskong) [inline sdcm/es.py]: Will the old info still exist with update?  can we just remove exist one and recreate a new one?

**Comment by bentsi** [comment]
@fruch @dkropachev 
1. the commit message should have a meaningful explanation
2. you didn't address @amoskong comment

File: sdcm/es.py (hunk 1 above)
Comment (dkropachev) [reply sdcm/es.py]: It is up to elastic.
I think it is pretty handy to be able to see prior test stats/errors.

**Comment by dkropachev** [comment]
I've updated comit message, but github UI does not pick it up

**Pull request merged**